streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueStore.java

```diff
@@ -52,10 +52,6 @@ public InMemoryKeyValueStore(final String name,
         this.map = new TreeMap<>();
     }
 
-    public KeyValueStore<K, V> enableLogging() {
```

Comment:
Does this mean we won't have logging for in-memory key-value stores?

Reply:
No. This is old code before the state store refactoring via KIP-182. Both, persistent and in-memory, key-value store are wrapped with `ChangeLoggingKeyValueBytesStore` since then, and `enableLogging` is not called anywhere anymore, and thus the class is not used any longer.

Reply:
:+1: Thanks for the catch.